src/i18n/es.js

```diff
@@ -119,7 +119,7 @@ export const messages_es = {
   'editor.all_sides.label': `Todos Lados`,
   'editor.alternate_text.label': `Texto Alternativo`,
   'editor.anchor.section_already_exists': `Ya existe una sección de página con este nombre`,
-  'editor.background_color.label': `Color Fondo`,
+  'editor.background_color.label': `Color fondo`,
```

Comment:
Que opionás de:
 
```suggestion
  'editor.background_color.label': `Color del fondo`,
```

(lo mismo en las demás entradas)

CC: @GGossweiler

Reply:
Nos les parece mas usado **_color de fondo_**

Reply:
Osea:

```suggestion
  'editor.background_color.label': `Color de fondo`,
```

Reply:
Sí! Faltan casi todos los artículos. Color del texto, Color del fondo, Ancho del contenido, Familia de la fuente. Si ven alguno más sospechoso me avisan.

Reply:
Según [lo hablado](https://makingsense.slack.com/archives/CLBEN4JAV/p1678994237063369?thread_ts=1678992624.890959&cid=CLBEN4JAV) con @GGossweiler:

```suggestion
  'editor.background_color.label': `Color del fondo`,
```